index.html

```diff
@@ -0,0 +1,247 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" media="screen" href="./styles.css">
+    </head>
+
+    <body>
+        <section id="portfolio" class="project-section">
+            <div>
+                <div class="projects">
+                    <div class="img-web">
+                        <img class="" src="images/project1.png" alt="project1">
+                    </div>
+                    <div class="project-description">
+                        <h1 class="title">Tonic</h1>
+                        <div class="role">
+                            <div class="role-name">CANOPY</div>
+                            <div class="role-dot"></div>
+                            <div class="role-job">Back End Dev</div>
+                            <div class="role-dot"></div>
+                            <div class="role-year">2015</div>
+                        </div>
+                        <div class="project-details">
```

Comment:
- [ ] It is required to use a display:grid for the main container of the projects.